src/plasma/status.h

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
```

Comment:
I don't know if it matters, but some of this code does not come from the LevelDB authors.